### PR TITLE
Docs improvements for  wait feature.

### PR DESCRIPTION
**Pull request by alexsomesan**
### Description

Add additional details on using the `wait` feature of `kubernetes_manifest`

### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch?

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### Release Note
Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-kubernetes/blob/main/CHANGELOG.md):

```release-note
Add additional details on using the wait feature of kubernetes_manifest
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

File: website/docs/r/manifest.html.markdown [hunk 1]
```diff
@@ -232,7 +233,26 @@ The following arguments are supported:
 
 - `rollout` (Optional) When set to `true` will wait for the resource to roll out, equivalent to `kubectl rollout status`. 
 - `condition` (Optional) A set of condition to wait for. You can specify multiple `condition` blocks and it will wait for all of them. 
-- `fields` (Optional) A map of fields and a corresponding regular expression with a pattern to wait for. The provider will wait until the field matches the regular expression. Use `*` for any value. 
+- `fields` (Optional) A map of field paths and a corresponding regular expression with a pattern to wait for. The provider will wait until the field's value matches the regular expression. Use `*` for any value.
+
+A field path is a string describing the fully qualified address of a field within the resource, including its parent fields all the way up to "object". The syntax of a path string follows the rules below:
```
Comment (BrianMMcClain) [inline website/docs/r/manifest.html.markdown:238]:
```suggestion
A field path is a string that describes the fully qualified address of a field within the resource, including its parent fields all the way up to "object". The syntax of a path string follows the rules below:
```

File: website/docs/r/manifest.html.markdown [hunk 2]
```diff
@@ -232,7 +233,26 @@ The following arguments are supported:
...
+  * Fields of objects are addressed with `.`
+  * Keys of a map field are addressed with `["<key-string>"]`
+  * Elements of a list or tuple field are addresed with `[<index-numeral>]`
+
+  For example, to wait for a ServiceAccount token to be created in a Secret, where the `data` field of Secret is a map, do the following:
```
Comment (BrianMMcClain) [inline website/docs/r/manifest.html.markdown:243]:
```suggestion
  The following example waits for Kubernetes to create a ServiceAccount token in a Secret, where the `data` field of the Secret is a map.
```

File: website/docs/r/manifest.html.markdown [hunk 3]
```diff
@@ -232,7 +233,26 @@ The following arguments are supported:
...
+  ```hcl
+  wait {
+    fields = {
+      "data[\"token\"]" = ".+"
+    }
+  }
+  ```
+  If unsure which type a certain field is, the [`type()`]() Terraform function can determine it. With the resource created and present in state, run `terraform console` and then the following command:
```
Comment (BrianMMcClain) [inline website/docs/r/manifest.html.markdown:251]:
```suggestion
  You can use the [`type()`]() Terraform function to determine the type of a field. With the resource created and present in state, run `terraform console` and then the following command:
```

**Review by BrianMMcClain — APPROVED** [review]
Just a couple of small tweaks, otherwise LGTM!

**Review by jrhouston — APPROVED** [review]

**Pull request merged**